jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/ContentResolver.java

```diff
@@ -51,6 +51,8 @@ public ContentResolver(JsonFactory jsonFactory) {
                 .enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS);
     }
 
+    static JsonNode cachedStdIn;
```

Comment:
Could you explain why you introduced a static member here?

Why not just:

```java
            if(uri.getPath().endsWith("/-")) {
                return objectMapper.readTree(System.in);
            } else {
                return objectMapper.readTree(uri.toURL());
            }
```

Reply:
Because its called multiple times when dealing with a Schema. Can only read the input once.